components/SlideEach/SlideEach.tsx

```diff
@@ -0,0 +1,21 @@
+import { SlideIn } from "components/SlideIn";
+import { ISlideIn } from "components/SlideIn/SlideIn";
+import React, { FC, ReactNode } from "react";
+
+interface ISlideEach extends Omit<ISlideIn, "children"> {
+	children: ReactNode;
+}
+
+export const SlideEach: FC<ISlideEach> = ({ children, ...slideProps }) => {
+	return (
+		<>
+			{React.Children.map(children, (child, i) => {
+				return (
+					<SlideIn key={`slide-in=${i}`} {...slideProps}>
```

Comment:
its slide-in- (not slide-in=)

Reply:
Ok really sorry but it works correctly 😂

Reply:
resolved